homeassistant/components/analytics/analytics.py

```diff
@@ -65,11 +63,16 @@ def onboarded(self) -> bool:
         """Return bool if the user has made a choice."""
         return self._data.get(ATTR_ONBOARDED, False)
 
+    @property
+    def supervisor(self) -> bool:
+        """Return bool if a supervisor is present."""
+        return hassio.DOMAIN in self.hass.data
```

Comment:
Use this:
https://github.com/home-assistant/core/blob/6fc3406c938eb6f88ea057603a9346ec2aba3721/homeassistant/components/hassio/__init__.py#L316